### ISSUE TITLE
Can't import any stdlib 

### ISSUE THREAD

**Issue by brunobord** [sirredbeard/nim_lang_snap#11]
result of `snap --version`:

```
 snap --version
snap    2.45.2
snapd   2.45.2
series  16
ubuntu  18.04
kernel  4.15.0-111-generic
```

Installed the classic nim snap using:

```sh
snap install --classic nim-lang
```

I've then tried to compile the following example:

```nim
import strformat

let msg = "hello"
doAssert &"{msg}\n" == "hello\\n"
```

It failed to compile:

```
nim c people.nim 
Hint: system [Processing]
Hint: widestrs [Processing]
Hint: io [Processing]
Hint: people [Processing]
/home/user/Nim/example.nim(1, 8) Error: cannot open file: strformat
```

I've also tried other examples, using different imports, and they've all failed at the import line(s).

Do you have any clue on why this is failing and how to fix this?

**Comment by wiml** [comment]
I'm having the same problem. Passing `--lib:/snap/nim-lang/539/lib` doesn't help. Passing  (for example) `-p:/snap/nim-lang/539/lib/pure` helps for imports of files in that particular directory, but I would need to do it for each subdirectory of `lib`.

```
$ snap --version
snap    2.45.3.1
snapd   2.45.3.1
series  16
ubuntu  19.10
kernel  5.3.0-64-generic
```

And `snap info nim-lang` says (*inter alia*):

```
snap-id:      yx0ma3S2U2Xjk80P61COaqrXdLV7IsgZ
tracking:     latest/stable
refresh-date: yesterday at 14:43 PDT
channels:
  latest/stable:    1.2.0 2020-06-07 (539) 12MB classic
  latest/candidate: ↑                           
  latest/beta:      ↑                           
  latest/edge:      ↑                           
installed:          1.2.0            (539) 12MB classic
```

**Comment by narimiran** [comment]
@sirredbeard this is still an issue. Any idea if/when it will get fixed?

**Comment by sirredbeard** [comment]
I have been investigating this. An strace output of a build shows nim-lang.nim properly locating and parsing the /lib/ folder within the snap.

However, at some point, it goes to look for /snap/nim-lang/current/lib/strformat.nim. strformat.nim isn't in that folder though, it's in /pure/ as you point out, and it fails.

Setting --lib or --path doesn't appear to fix this for me. Neither does adding the path to /pure (and all other /lib subdirectories) to $PATH. /lib/ is shown in `nim dump`, but none of the subdirectories.

I investigated linking all individual .nim files into /lib/ but there are at least 5-6 duplicate named that would collide.

Then I remembered nim.cfg. I had left out /config/nim.cfg in the build.
